index.d.ts

```diff
@@ -0,0 +1,81 @@
+// JsonML
+// see: http://www.jsonml.org/syntax/
+//
+// `Node` type imported from lib.dom
+//
+/// <reference lib="dom"/>
```

Comment:
Attempting to mark that `lib.dom` is required. The `Node` type is available as a global after adding `'dom'` to `tsconfig`. I wish I could reference it more explicitly.

Overall, I'm not sure if I'm handling namespacing/scope propertly.